core/src/main/java/org/kiwix/kiwixmobile/core/help/HelpActivity.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2019 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package org.kiwix.kiwixmobile.core.help
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import androidx.core.net.toUri
+import androidx.recyclerview.widget.DividerItemDecoration
+import org.kiwix.kiwixmobile.core.R
+import org.kiwix.kiwixmobile.core.base.BaseActivity
+import org.kiwix.kiwixmobile.core.di.components.CoreComponent
+import org.kiwix.kiwixmobile.core.utils.Constants
+import org.kiwix.kiwixmobile.core.utils.LanguageUtils.Companion.getCurrentLocale
+import kotlinx.android.synthetic.main.activity_help.activity_help_feedback_image_view
+import kotlinx.android.synthetic.main.activity_help.activity_help_feedback_text_view
+import kotlinx.android.synthetic.main.activity_help.activity_help_recycler_view
+import kotlinx.android.synthetic.main.layout_toolbar.toolbar
+import java.util.HashMap
+
+class HelpActivity : BaseActivity() {
+  private val titleDescriptionMap by lazy {
+    listOf(
+      R.string.help_2 to R.array.description_help_2,
+      R.string.help_5 to R.array.description_help_5
+    )
+  }
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_help)
+
+    activity_help_feedback_text_view.setOnClickListener { sendFeedback() }
+    activity_help_feedback_image_view.setOnClickListener { sendFeedback() }
+    setSupportActionBar(toolbar)
+    toolbar.setNavigationOnClickListener { onBackPressed() }
+    supportActionBar?.let {
+      it.setDisplayHomeAsUpEnabled(true)
+      it.setTitle(R.string.menu_help)
+    }
+    activity_help_recycler_view.addItemDecoration(
+      DividerItemDecoration(this, DividerItemDecoration.VERTICAL)
+    )
+    activity_help_recycler_view.adapter = HelpAdapter(
+      titleDescriptionMap.associate {
+        getString(it.first) to resources.getStringArray(it.second)
+          .joinToString(separator = System.getProperty("line.separator")!!)
+      } as HashMap<String, String>?
```

Comment:
I don't want to see this cast to a HashMap, yo might have to change the type in HelpAdapter to just Map

Reply:
Ok